packages/driver/cypress/integration/e2e/multi-domain/basic_login_spec.ts

```diff
@@ -1,25 +1,11 @@
-// @ts-ignore / session support is needed for visiting about:blank between tests
+// @ts-nocheck / session support is needed for visiting about:blank between tests
```

Comment:
Do we still need the `@ts-ignore`s for the file if we plan to use `@ts-nocheck` or is this a temporary thing while this is under construction?

Reply:
we don't need the ignores if there is the no-check, i will remove them